src/Adapter/Chain/CachePoolChain.php

```diff
@@ -326,7 +327,8 @@ private function handleException($poolKey, $operation, CachePoolException $excep
 
         $this->log(
             'warning',
-            sprintf('Removing pool "%s" from chain because it threw an exception when executing "%s"', $poolKey, $operation),
+            sprintf('Removing pool "%s" from chain because it threw an exception when executing "%s"', $poolKey,
+                $operation),
```

Comment:
:) ... i use code formater in PhpStorm with psr2 rule, and it have 120 line lenght rule.
If you want, discard this change
:)